data_collection/advertisements/powerblade_adv_influx.js

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env node
+// original file: https://github.com/lab11/powerblade/tree/master/data_collection/advertisements
+
+var noble = require('noble');
+var config = require('./config');
```

Comment:
Can you add a description of what this config file should look like to the README.md file in this folder? That will allow other people in the future to use this script.

Reply:
Added